core/tests/batch_loader_tests.py

```diff
@@ -85,8 +85,10 @@ def test_load_batch(self):
             'uuml_thys_ocr.txt')
         self.assertEqual(solr_doc['ocr_eng'], file(f).read().decode('utf-8'))
 
-
         # purge the batch and make sure it's gone from the db
         loader.purge_batch('batch_uuml_thys_ver01')
         self.assertEqual(Batch.objects.all().count(), 0)
         self.assertEqual(Title.objects.get(lccn='sn83045396').has_issues, False)
+
+
```

Comment:
We should remove the extra lines here following PEP-8

Reply:
Removed the extra lines